NT_Client_Subscribe_Node:
  ros__parameters:
    NT_server_ip: "10.80.20.2"
    sampling_time: 0.1
    
    sub_NT_names: []  # the NT names you want to subscribe, like "/data/test_vel"
    msg_types: [] # the corresponding ROS msg type, like "geometry_msgs/msg/Twist"
    pub_rostopic_names: []  # the ROS topic name to be published, like "/restored_cmd_vel"